internal-export-file/export-file-stix/src/export-file-stix.py

```diff
@@ -49,51 +50,89 @@ def _process_message(self, data):
                 + ") to "
                 + file_name
             )
-        else:
-            list_params = data["list_params"]
-            self.helper.log_info(
-                "Exporting list: "
-                + entity_type
-                + "/"
-                + export_type
-                + " to "
-                + file_name
-            )
-            bundle = self.helper.api_impersonate.stix2.export_list(
-                entity_type,
-                list_params["search"],
-                list_params["filters"],
-                list_params["orderBy"],
-                list_params["orderMode"],
-                max_marking,
-                list_params.get("types"),
-                list_params.get("elementId"),
-                list_params.get("fromId"),
-                list_params.get("toId"),
-                list_params.get("elementWithTargetTypes"),
-                list_params.get("fromTypes"),
-                list_params.get("toTypes"),
-                list_params.get("relationship_type"),
-            )
+
+        else:  # export_scope = 'query' or 'selection'
+            if "element_id" in data:
+                element_id = data["element_id"]
+            else:
+                element_id = None
+
+            if export_scope == "selection":
+                selected_ids = data["selected_ids"]
+                list_filters = "selected_ids"
```

Comment:
It's normal to assign a specific string here ?

Reply:
no it's not, I assigned a value because list_filters should have a value to call push_list_export(file_name, data, list_filters).... :(